AppCenterDataStorage/AppCenterDataStorage/Internal/Ingestion/MSCosmosDbIngestion.m

```diff
@@ -0,0 +1,48 @@
+#import "MSCosmosDbIngestion.h"
+#import "MSAppCenter.h"
+#import "MSAppCenterInternal.h"
+#import "MSHttpIngestionPrivate.h"
+#import "MSLoggerInternal.h"
+
+@implementation MSCosmosDbIngestion
+
+- (instancetype)init {
+  // @TODO: clean up once the MSHttpIngestion is refactored to take generic requests.
```

Comment:
Usually comments have an empty line above but as this is a TODO, it's ok to not have it.